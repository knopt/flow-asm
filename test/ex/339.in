4
13
5.661323234486947
2.774717397308182 6.303179854408558 -8.494195454377941 7.5826054621536585 
0.4755305849901337 4.155726872374245 -8.40636075553763 5.199253348610833 
9.060400021897308 -9.87802255355869 1.2290125001931056 3.595646511477149 
-7.9063115748910695 -9.711573435147447 5.37681891648959 -0.6000240378334158 
-8.969221738766315 -1.8251089946999883 2.5225907455864593 -0.9710183820132023 
-2.154334852377451 4.241698808763065 -4.38188491470423 1.5302295198894456 
8.591318260759124 -8.18641787557481 -5.650197336434124 5.363706053890461 
-1.3908563790564017 -5.732382791876667 -3.149111704582614 9.554705690153359 
-2.0384488490612203 -2.7921263116196293 1.5393080001937243 -9.513691599945117 
-3.9110288013348615 4.026595699480662 4.22500350651098 -8.193352433361024 
-8.371703513161496 -2.672250885440688 1.0720433731464123 -9.47221342863211 
-5.146683307702951 -6.606992195217078 9.463965499979292 1.25664322217024 
8.436661195952631 -0.6587847806824101 -7.097809438385871 -1.446004762862831 

9.106614078288718 2.460592403568583 -9.369649475994473 8.164303931587785 0.3890528159345976 -6.243878025500371 -1.2625180246435104 -3.317864238237087 5.946871985197713 6.574480015621695 -2.6907713477564403 -7.803478568657633 -6.721402173071422 
