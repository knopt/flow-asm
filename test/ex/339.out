23.826828349120873 -58.82221830954725 147.39108899491467 -187.44499374283384 
66.4583746020511 -64.0226210759878 191.70729056357877 -167.41199247862593 
-282.37056062638914 247.50772726881158 -138.08751004347073 -82.34059400325773 
211.78950825815673 154.69100198831697 -246.81925864159967 82.92244278803373 
201.0372295618325 -88.86341231854622 -104.55003453802354 51.69526474260296 
16.39458056068686 -186.84075066840876 69.28319595287877 -59.43062411627038 
-315.9499399177403 243.67188308189304 56.867162229818916 -158.33226081775558 
82.81363316152121 123.60271368185823 -31.887447888933288 -325.5023030910708 
77.75137365874859 25.030505807249142 -61.406463137261014 282.2992771282202 
103.51581953427116 -221.96514450155837 -108.72659067454941 154.95019951772616 
155.13317411593633 -40.31188501581025 18.48686672027881 302.9500057202316 
43.44288091231718 132.78723896593635 -348.80507084420583 -76.66147811551022 
-246.21841956621805 -8.248694189631408 125.89582678925373 43.62273407971344 
